### PR TITLE
feat(workflow/projectCreation): Add default value to event threshold

### PR DESCRIPTION
**Pull request by jeffkwoh**
Having a placeholder value is confusing as the user expects the value to be a default pre-filled value. This PR changes the placeholder value to a default value that can be used to submit forms.

See: https://www.notion.so/sentry/Default-Alert-Options-450bbbc6f83b4ba29350fb9a59cf5746

### CODE REVIEW

File: src/sentry/static/sentry/app/views/projectInstall/issueAlertOptions.tsx [hunk 1]
```diff
@@ -110,8 +106,7 @@ class IssueAlertOptions extends AsyncComponent<Props, State> {
       alertSetting: `${Actions.CUSTOMIZED_ALERTS}`,
       metric: MetricValues.ERRORS,
       interval: '',
-      placeholder: '',
-      threshold: '',
+      threshold: '10',
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/projectInstall/issueAlertOptions.tsx]: mmm I would pull this out into a local const so it's more clear this is our default value we've chosen

File: src/sentry/static/sentry/app/views/projectInstall/issueAlertOptions.tsx [hunk 2]
```diff
@@ -144,7 +139,6 @@ class IssueAlertOptions extends AsyncComponent<Props, State> {
             min="0"
             name=""
             value={this.state.threshold}
-            placeholder={this.state.placeholder}
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/projectInstall/issueAlertOptions.tsx]: Why was placeholder previously in state? Where was it coming from - we should probably still have fixed value for the placeholder (i.e. if they delete the default value, we should still show something)

File: src/sentry/static/sentry/app/views/projectInstall/issueAlertOptions.tsx (hunk 1 above)
Comment (jeffkwoh) [reply src/sentry/static/sentry/app/views/projectInstall/issueAlertOptions.tsx]: Done.

File: src/sentry/static/sentry/app/views/projectInstall/issueAlertOptions.tsx (hunk 2 above)
Comment (jeffkwoh) [reply src/sentry/static/sentry/app/views/projectInstall/issueAlertOptions.tsx]: Initially the placeholder value is fetched from the end point. Added the default value as a placeholder value.

**Review by billyvg — APPROVED** [review]

**Pull request merged**